test/verify/machineslib.py

```diff
@@ -1343,32 +1364,28 @@ def allowBugErrors(location, original_exception):
 
             error_location = ".modal-footer div.alert"
 
-            if ui_validation:
-                b.wait_present(error_location)
-                waitForError(errors, error_location)
-            else:
-                b.wait_present(".modal-footer .spinner")
-                b.wait_not_present(".modal-footer .spinner")
-                try:
-                    with b.wait_timeout(10):
-                        b.wait_present(error_location)
-                        waitForError(errors, error_location)
-
-                    # dialog can complete if the error was not returned immediately
-                except Exception as x1:
-                    if b.is_present("#create-vm-dialog"):
-                        # allow CPU errors in the dialog
-                        allowBugErrors(error_location, x1)
-                    else:
-                        # then error should be shown in the notification area
-                        error_location = "#notification-area-notification-1 div.notification-message"
-                        try:
-                            with b.wait_timeout(20):
-                                b.wait_present(error_location)
-                                waitForError(errors, error_location)
-                        except Exception as x2:
-                            # allow CPU errors in the notification area
-                            allowBugErrors(error_location, x2)
+            b.wait_present(".modal-footer .spinner")
+            b.wait_not_present(".modal-footer .spinner")
+            try:
+                with b.wait_timeout(10):
+                    b.wait_present(error_location)
+                    waitForError(errors, error_location)
+
+                # dialog can complete if the error was not returned immediately
+            except Exception as x1:
```

Comment:
Please catch `testlib.Error` here, not a blanket exception. (Once more below)

Reply:
Oh that was just huge  whitespace change. Didn't introduce this here.
But I 'll post a commit on top of this one to fix this all these plan Exception catches in this file.